docs/azure_arc_jumpstart/azure_arc_servers/day2/arc_monitoring/_index.md

```diff
@@ -0,0 +1,235 @@
+
+   ![Screenshot showing Map Insights on Azure Arc-enabled server](./26.png)
+
+### Azure Dashboard and Workbooks
+
+- Locate the **Shared Dashboard** that was deployed in your resource group. Click on its **name** and then on the **Go to dashboard** link:
+
+   ![Screenshot showing shared dashboard in resource group](./27.png)
+
+   ![Screenshot showing Go to dashboard button](./28.png)
+
+- The goal of this dashboard is to provide a quick overview of the current status of the monitored Azure Arc-enabled servers. You can play with the dashboards controls **Auto refresh** and **UTC Time**. Note that this dashboard is made up of three tiles, each of them being a quick access button and a summary of the deployed Azure workbooks:
+  - **OSPerformanceAndCapacity:** provides a quick performance overview of the Azure Arc-enabled servers. In addition, it contains several charts with more insights of some key OS performance counters.
+  - **AlertsConsole:** console that shows the alerts that have been fired within a time range.
+  - **WindowsEvents:** Windows Events central visualizer.
+
+   ![Screenshot showing Azure Dashboard summary](./29.png)
+
+- To get into one of the pinned workbooks, like for example the **OSPerformanceAndCapacity** workbook, just click on the **Top servers** tile:
+
+   ![Screenshot showing how to get into the OSPerformanceAndCapacity workbook](./30.png)
+
+- This workbook, apart from the **Top servers** summary table available at the top of the **shared dashboard**, shows two charts for each of some the key OS performance counters:
+
+   ![Screenshot showing performance charts of the OSPerformanceAndCapacity workbook](./31.png)
+
+- In addition, at the bottom of this workbook, there are **two additional charts**. One of them shows which are the **processes that use more CPU for Windows machines**. The second one shows the **network usage by process per each Azure Arc-enabled server**:
+
+   ![Screenshot showing cpuusage and networkusage by processes charts of the OSPerformanceAndCapacity workbook](./32.png)
+
+- On the other hand, the **WindowsEvents workbook**, accessible from the **Shared Dashboard**, provides additional insights of the Windows Events collected. It shows the number of events by EventLog and Severity, along with its hourly distribution:
+
+   ![Screenshot showing top charts of the WindowsEvents workbook](./33.png)
+
+- At the bottom of the **WindowsEvents workbook**, click on one of the computers and its severity to load the table below, where all details per event are provided:
+
+   ![Screenshot showing top charts of the WindowsEvents workbook](./34.png)
+
+## Clean up environment
+
+To delete the entire deployment, simply delete the resource group from the Azure portal.
+
+![Screenshot showing how to delete resource group](./35.png)
+
+## Acknowledgment
```

Comment:
I appreciate the work contributed by Diego. 

Unfortunately an Acknowledgment section is not a convention we have used previously for Jumpstart contributions and we need to remove this.